editor/property_selector.cpp

```diff
@@ -338,46 +338,24 @@ void PropertySelector::_item_selected() {
 	}
 	String name = item->get_metadata(0);
 
-	String class_type;
-	if (type != Variant::NIL) {
-		class_type = Variant::get_type_name(type);
-
-	} else {
-		class_type = base_type;
-	}
-
 	DocData *dd = EditorHelp::get_doc_data();
 	String text;
 
 	if (properties) {
-		String at_class = class_type;
-
-		while (at_class != String()) {
```

Comment:
The removal of the `at_class` "parse all parents upwards" logic will break things for custom classes, no?

Reply:
Provided that custom classes are added properly to the class_list, then i don't think so.